setup.sh

```diff
@@ -221,7 +222,9 @@ then
 		if [ $? -eq 0 ]
 		then 
 		 echo 'Deleting existing hadoop folder "'hadoop-${hadoopver}'" from '$i' '| tee -a $log
+		 echo 'Deleting existing hadoop folder "'hadoop-${hadoopver}.tar.gz'" from '$i' '| tee -a $log
```

Comment:
Leaving out existing tar.gz files is done on purpose to minimize the overall time taken when setup scripts are re-executed.